cloud-sql/postgres/sqlalchemy/main.py

```diff
@@ -109,11 +124,12 @@ def init_unix_connection_engine(db_config):
     cloud_sql_connection_name = os.environ["CLOUD_SQL_CONNECTION_NAME"]
 
     pool = sqlalchemy.create_engine(
+
         # Equivalent URL:
         # postgres+pg8000://<db_user>:<db_pass>@/<db_name>
         #                         ?unix_sock=<socket_path>/<cloud_sql_instance_name>/.s.PGSQL.5432
         sqlalchemy.engine.url.URL(
-            drivername="postgres+pg8000",
+            drivername="postgresql+pg8000",
```

Comment:
are we sure this is right? if so, we need to update the URL in the comments as well

Reply:
SQLAlchemy was spitting out lots of warnings about the driver name `"postgres"` being deprecated.